GameEngine/GameEngine/Engine/Graphic/DRBCameraDrawableList.hpp

```diff
@@ -0,0 +1,55 @@
+#pragma once
+
+#include <list>
+#include <memory>
+#include <glm/glm.hpp>
+
+namespace AGE
+{
+	struct DRBData;
+	class TextureCubeMap;
+
+	struct DRBSpotLightDrawableList
+	{
+		std::list<std::shared_ptr<DRBData>> meshs;
+		std::shared_ptr<DRBData> spotLight;
+	};
+
+	struct CameraData
+	{
+		bool                            activated;
+		std::shared_ptr<TextureCubeMap> texture;
+		glm::mat4                       projection;
+		unsigned int                    pipeline;
+
+		CameraData(CameraData const &cam);
+		CameraData();
+		CameraData &operator=(CameraData const &cam);
+	};
+
+	struct CameraInfos
+	{
+		glm::mat4 view;
+		CameraData data;
+	};
+
+	struct DRBCameraDrawableList
+	{
+		std::list<std::shared_ptr<DRBData>> meshs;
```

Comment:
No because we will replace that list by a lock free list that will be filled by multiple threads in parallel :)